reliability/config/enhance_reliability.yaml

```diff
@@ -18,23 +19,64 @@ reliability:
     - template: django-psql-persistent
     - template: rails-pgsql-persistent
     - template: dancer-mysql-persistent
-
   users:
     - admin_user: kubeadmin
     - admin_password: <password>
     - user_file: /Users/qili/Downloads/users.spec
```

Comment:
can we make this similar to the password, to be more generic

ex.) <user.spec>

Reply:
Thanks for the comment Paige. Fixed in the latest commit.